### ISSUE TITLE
https curl request to plugins API fails

### ISSUE THREAD

**Issue by Thykof** [massalabs/station#1122]
**The problem(s)**

This curl request fails:

```
curl --location 'http://station.massa/plugin/massa-labs/massa-wallet/api/accounts/nathan/sign' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--data '{"operation":"VGVzdA=="}'
```

Postman error: `SSL Error: Unable to verify the first certificate`

Massa Station logs:

```
2023-08-24T15:07:28.156+0200	DEBUG	logger/global.go:40	SNI - generating TLS certificate
2023/08/24 15:07:28 http: TLS handshake error from 127.0.0.1:57988: EOF
```

Hypothesys: the CA is not in the system store. However on Macos we implemented the copy of the root CA into the system store.

**MassaStation version**
The version of MassaStation the bug is present in.

v0.3.1, main branch

**Desktop (please complete the following information):**
 - OS: MacOS, Ubuntu, maybe Windows

**Comment by Thykof** [comment]
On ubuntu I have followed this tutorial and successfully performed a POST request with https https://learning.postman.com/docs/sending-requests/certificates/#adding-ca-certificates

**Comment by gregLibert** [comment]
Indeed. Furthermore, as MS is running on your local computer, CA being unverified is not such an important security vulnerability.
